types/onfleet__node-onfleet/onfleet__node-onfleet-tests.ts

```diff
@@ -1,82 +1,79 @@
 import Onfleet = require('@onfleet/node-onfleet');
 
 const onfleet = new Onfleet('test-api-key');
-const isValid = onfleet.verifyKey();
+const isValid = onfleet.verifyKey().then().catch();
```

Comment:
For consistency, you can do `const isValid = await onfleet.verifyKey();`

Reply:
I would do this if I could but JS doesn't allow you to do `await` in the global scope — always has to be inside of an `async` function.